aptos-move/e2e-tests/goldens/language_e2e_testsuite__tests__on_chain_configs__initial_aptos_version_version_4.exp

```diff
@@ -10,6 +10,12 @@ Ok(
                             ),
                             Modification(010000000000000000a493d600000000020000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000001),
                         ),
+                        (
+                            AccessPath(
+                                AccessPath { address: 0000000000000000000000000000000000000000000000000000000000000001, path: 010000000000000000000000000000000000000000000000000000000000000001057374616b651456616c696461746f72506572666f726d616e636500 },
+                            ),
+                            Modification(0101000000000000000000000000000000),
```

Comment:
hmm, I wonder why we have one more write here

Reply:
I assume it is because proposer and proposer_index didn't match in aptos-move/e2e-tests/src/executor.rs (i.e. proposer_index was None, skipping updating stats.